### PR TITLE
activate integration tests for Pharo5

### PR DESCRIPTION
**Pull request by woglinde**

### CODE REVIEW

File: docker/docker-image.sh [hunk 1]
```diff
@@ -1,7 +1,12 @@
 #!/bin/bash
 
-if [ x"${TRAVIS_SMALLTALK_VERSION}" != x"Pharo-4.0" ]; then
-    exit 0
+set -u
+
+if [ x"${TRAVIS_SMALLTALK_VERSION}" = x"Pharo-4.0" ]; then
```
Comment (zecke) [inline docker/docker-image.sh]: What about making this symmetric and have Pharo-4.0 and Pharo-5.0 behave the same? Copy the VM in both cases. Could you then fold it into a single Dockerfile as well?

File: docker/Dockerfile.osmo-smsc5 [hunk 2]
```diff
@@ -0,0 +1,35 @@
+FROM debian
```
Comment (zecke) [inline docker/Dockerfile.osmo-smsc5]: osmo-smsc4 and osmo-smsc5 sound like different SMSC versions. If you copy the VM in both cases then there should be no difference in this file and the only difference will be in docker/docker-image.sh?

**Comment by woglinde** [comment]
I switched back to a single Dockerfile and use it for pharo-4 and pharo-5.

File: docker/Dockerfile.osmo-smsc [hunk 3]
```diff
@@ -10,6 +10,15 @@ ADD ./ /tmp/osmo-smsc
 
 RUN make -C /tmp/osmo-smsc/ install-rdepends install && rm -rf /tmp/osmo-smsc
 
+# reuse the start skript
```
Comment (zecke) [inline docker/Dockerfile.osmo-smsc]: script. You said I was not as picky as I used to be. Please rebase quickly

File: docker/Dockerfile.osmo-smsc [hunk 4]
```diff
@@ -10,6 +10,15 @@ ADD ./ /tmp/osmo-smsc
...
+RUN cp /usr/bin/pharo-vm-nox /usr/bin/pharo-vm-nox.bak && \
+    dpkg --purge pharo-vm-core && \
+    mv /usr/bin/pharo-vm-nox.bak /usr/bin/pharo-vm-nox
```
Comment (zecke) [inline docker/Dockerfile.osmo-smsc]: ah fun. we could just pass --nodisplay and then use the "pharo" binary. But that is fine as well.

**Comment by woglinde** [comment]
I have fixed the typo.

**Pull request merged**